data/src/main/java/com/my/kizzy/data/rpc/CommonRpc.kt

```diff
@@ -12,12 +12,12 @@
 
 package com.my.kizzy.data.rpc
 
-import kizzy.gateway.entities.presence.Timestamps
-
 data class CommonRpc(
     val name: String = "",
     val details: String? = "",
     val state: String? = "",
+    val party1: Int? = null,
+    val party2: Int? = null,
```

Comment:
Maybe change the name `party1` & `party2` to `party` & `partyMaxSize` as its confusing

Reply:
Changing `party1` to `partyCurrentSize` would make more sense in that direction also, sure!